src/routes/services.ts

```diff
@@ -31,16 +32,19 @@ export const vpn = {
 				captureException(err, 'Error authenticating device for VPN', { req });
 				res.status(500).send(err);
 			}),
-	clientConnect: (req: Request, res: Response) => {
+	clientConnect: (req: Request, res: Response): Promise<void> => {
```

Comment:
The real response type is `void | Promise<void>` - which is why I think you've done the weird type casting below